notebooks/04_model_select_and_optimize/aml_hyperparameter_tuning.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<p>how can this be extended to other models? I wonder if this notebook is more focused on TF models or is generic</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
As mentioned earlier, that is good topic to discuss. Since different models have different set of hyper-params, it is not possible to generalize 100%, but at least can modularize the hyperparameter parsing part so that can reuse other scripts as much as possible.

We will have more clear idea about how much of the codes can be generalized once we start adding more AzureML examples.